terraform/main_vm_start.sh

```diff
@@ -40,10 +42,10 @@ snap install --classic --channel=1.14/stable go
 export GOPATH=$HOME/go
 export PATH=/snap/bin:$GOPATH/bin:$PATH
 
-/snap/bin/go get -v -t github.com/Alexandr-TS/fleetspeak/...
+/snap/bin/go get -v -t github.com/Alexandr-TS/fleetspeak/... ||:
 
 cd $HOME/go/src/github.com/Alexandr-TS/fleetspeak/
-git checkout tmp_prep_cloud
+git checkout prep_cloud
```

Comment:
After this PR is approved, these lines should be replaced with
`/snap/bin/go get -v -t github.com/google/fleetspeak/...`
`cd /go/src/github.com/google/fleetspeak/`